modules/installation-configuration-parameters.adoc

```diff
@@ -1331,6 +1331,22 @@ Additional GCP configuration parameters are described in the following table:
 |====
 |Parameter|Description|Values
 
+|`controlPlane.platform.gcp.osimage.project`
```

Comment:
Shall `osimage.project` be `osImage.Project` instead? FYI the [PR](https://github.com/openshift/installer/pull/7215/).